examples/src/HooksExample/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from "react";
```

Comment:
This example needs to actually use a hook. Or be removed.

Reply:
Better to leave it and use hooks